src/base/intl_support.c

```diff
@@ -3113,9 +3123,9 @@ intl_identifier_upper (const char *src, char *dst)
 int
 intl_identifier_fix (char *name, int ident_max_size, bool error_on_case_overflow)
 {
-  int i, length_bytes;
+  int name_length = 0, length_bytes;
```

Comment:
We usually use length when referring to characters and size when referring to raw bytes.
I propose to rename :
length_bytes -> original_size
name_length  -> truncated_size